compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -1095,14 +1095,21 @@ static LogicalResult setPackOpRootConfig(func::FuncOp entryPointFn, OpTy op) {
       DispatchLoweringPassPipeline::CPUDataTiling);
 }
 
-static LogicalResult setRootConfig(
-    func::FuncOp entryPointFn, IREE::LinalgExt::UnPackOp op,
+template <typename OpTy>
+static LogicalResult setUnPackOpRootConfig(
+    func::FuncOp entryPointFn, OpTy op,
     DispatchLoweringPassPipeline pipeline =
         DispatchLoweringPassPipeline::CPUDataTiling) {
+  // TODO(hanchung): Retire IREE::LinalgExt::UnPackOp. This is for having
+  // consistent configurations for unpack ops.
+  static_assert(
+      llvm::is_one_of<OpTy, IREE::LinalgExt::UnPackOp, tensor::UnPackOp>::value,
+      "applies to only pack operations");
```

Comment:
`unpack` ?

Reply:
Though my favorite choice is no message at all (if we can assume C++17) or empty string `""` (before C++17) as the assert condition here `llvm::is_one_of<OpTy, IREE::LinalgExt::UnPackOp, tensor::UnPackOp>::value` is easy to understand and the message does not add more useful information (and, in this case, was potentially inaccurate).

Reply:
yes, I like it better as well! removed the message.